test/e2e_node/criproxy_test.go

```diff
@@ -94,6 +98,133 @@ var _ = SIGDescribe(feature.CriProxy, framework.WithSerial(), func() {
 
 	})
 
+	ginkgo.Context("Image volume digest error handling", func() {
+		ginkgo.BeforeEach(func() {
+			e2eskipper.SkipUnlessFeatureGateEnabled(features.ImageVolume)
```

Comment:
are you doing this instead of adding WithFeatureGate to the test name because you want to execute it in some special test suite? I think it is best to follow the existing pattern